src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import duke.exceptions.DukeEmptyTodoDescriptionException;
+import duke.exceptions.DukeUnknownCommandException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DukeTest {
+
+    @Test
+    public void testWrongCommand() {
```

Comment:
I think that this method naming convention is fine, an possible alternative you and your team could use: is **featureUnderTest_testScenario_expectedBehavior()**